src/middleware/validation.schema.js

```diff
@@ -20,9 +20,7 @@ const followeeId = Joi.number()
   .integer()
   .required();
 
-const articleId = Joi.number()
-  .integer()
-  .required();
+const articleId = Joi.number().required();
 
```

Comment:
Please return the integer validation here as it was used to ensure that the number passed is am integer and not float

Reply:
ok